src/xcode/ENA/ENA/Source/Scenes/TicketValidation/FirstConsent/FirstTicketValidationConsentViewModel.swift

```diff
@@ -23,13 +23,14 @@ struct FirstTicketValidationConsentViewModel {
 		DynamicTableViewModel([
 			// Image with title
 			.section(
-				header: .image(
-					UIImage(named: "Illu_TicketValidation"),
-					title: AppStrings.TicketValidation.FirstConsent.title,
-					accessibilityLabel: AppStrings.TicketValidation.FirstConsent.imageDescription,
-					accessibilityIdentifier: AccessibilityIdentifiers.TicketValidation.FirstConsent.image
-				),
-				cells: []
+				cells: [
+					.headlineWithImage(
+						headerText: AppStrings.TicketValidation.FirstConsent.title,
+						image: UIImage(imageLiteralResourceName: "Illu_TicketValidation"),
+						accessibilityLabel: AppStrings.TicketValidation.FirstConsent.imageDescription,
+						accessibilityIdentifier: AccessibilityIdentifiers.TicketValidation.FirstConsent.image
+					)
+				]
```

Comment:
what makes the difference here?

Reply:
This type contains the gradient as a separate background view and pushes the illustration down if the text size of the header text is increased. The downside with this solution is, you need the illustration without the gradient in the correct size, I had to manually resize them and export them separately. But in the other solution they could overlap, that's why we invented this one when UX introduced this header style with the gradient in the background.